surf/src/diff.rs

```diff
@@ -274,7 +288,7 @@ impl Diff {
                                     let mut path = parent_path.borrow().clone();
                                     path.push(new_file_name.clone());
 
-                                    diff.add_modified_file(path, vec![]);
+                                    diff.add_modified_file(path, vec![], None);
```

Comment:
Actually, thanks for pointing this out! This is dead code :) I grepped for the usage of the `diff` that calls `collect_diff` and it's only used in the `examples` code.